app/controllers/sessions_controller.rb

```diff
@@ -18,8 +18,6 @@ def create
   end
 
   def destroy
-    session.destroy
-    redirect_to signin_path
```

Comment:
This part should have been left alone. The failing functionality was only around the redirect on login. It's sensible to crush the session and redirect a user to the login, most apps do something similar.